fs/glob.py

```diff
@@ -91,9 +172,95 @@ def imatch(pattern, path):
     except KeyError:
         levels, recursive, re_pattern = _translate_glob(pattern, case_sensitive=True)
         _PATTERN_CACHE[(pattern, False)] = (levels, recursive, re_pattern)
+    if path and path[0] != "/":
+        path = "/" + path
     return bool(re_pattern.match(path))
 
 
+def match_any(patterns, path):
+    # type: (Iterable[Text], Text) -> bool
+    """Test if a path matches any of a list of patterns.
+
+    Will return `True` if ``patterns`` is an empty list.
+
+    Arguments:
+        patterns (list): A list of wildcard pattern, e.g ``["*.py",
+            "*.pyc"]``
+        name (str): A filename.
```

Comment:
Says `name` here but `path` in the function definition. (Same "error" in `imatch_any` too.)